docs/releases/2.0.txt

```diff
@@ -258,6 +258,10 @@ Database backend API
   ``datetime_extract_sql()`` methods now return only the SQL to perform the
   operation instead of SQL and a list of parameters.
 
+* Add a ``DatabaseWrapper.display_name`` attribute with the name of the
```

Comment:
Sorry to revise my suggestion again: `Third-party database backends should add a...`